GitExtensions.sln

```diff
@@ -211,198 +206,297 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DeleteUnusedBranchesTests",
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EasyHook", "Externals\EasyHook\EasyHook\EasyHook.csproj", "{26B8A7CB-F911-4ADE-85F6-FF3FE95BE0AC}"
 EndProject
+Project("{840C416C-B8F3-42BC-B0DD-F6BB14C9F8CB}") = "Setup.AI.64", "Setup.AI.64\Setup.AI.64.aiproj", "{FDFCB87E-EBDC-450C-88E0-828754DCCDC1}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		MainBuild|Any CPU = MainBuild|Any CPU
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{F15A69AF-7EBD-4F69-A026-5071FA3EC61B}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{F15A69AF-7EBD-4F69-A026-5071FA3EC61B}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{F15A69AF-7EBD-4F69-A026-5071FA3EC61B}.MainBuild|Any CPU.ActiveCfg = Release|Any CPU
```

Comment:
Why add MainBuild configuration?

Reply:
I believe adding the Advanced Installer project did this. Given the initial idea was to use the GE project's output for the installers input files but cannot be used, a Visual Studio project for Advanced Installer is no longer necessary.

This is because the file structure is vastly different plus external dependencies are not mapped (and in different locations).

Building the installer will not be performed by the VS project nor Advanced Installer GUI but via command line. So in this case I'll remove this which should revert the Solution back.